inline.go

```diff
@@ -53,14 +53,45 @@ type QueryResponse struct {
 	// pagination. Offset length can’t exceed 64 bytes.
 	NextOffset string `json:"next_offset"`
 
-	// (Optional) If passed, clients will display a button with specified
-	// text that switches the user to a private chat with the bot and sends
-	// the bot a start message with the parameter switch_pm_parameter.
-	SwitchPMText string `json:"switch_pm_text,omitempty"`
-
-	// (Optional) Parameter for the start message sent to the bot when user
-	// presses the switch button.
-	SwitchPMParameter string `json:"switch_pm_parameter,omitempty"`
+	// (Optional) A JSON-serialized object describing a button to be shown
+	// above inline query results.
+	Button *QueryResponseButton `json:"button"`
+}
+
+// QueryResponseButton represents a button to be shown above inline query results.
+// You must use exactly one of the optional fields.
+type QueryResponseButton struct {
+	// Label text on the button
+	Text string `json:"text"`
+
+	// (Optional) Description of the Web App that will be launched when the
+	// user presses the button. The Web App will be able to switch back to the
+	// inline mode using the method switchInlineQuery inside the Web App.
+	WebApp *WebApp `json:"web_app"`
+
+	// (Optional) Deep-linking parameter for the /start message sent to the bot
+	// when a user presses the button. 1-64 characters, only A-Z, a-z, 0-9, _ and - are allowed.
+	Start string `json:"start_parameter"`
+}
+
+// InlineQueryChosenChat represents an inline button that switches the current
+// user to inline mode in a chosen chat, with an optional default inline query.
+type InlineQueryChosenChat struct {
```

Comment:
1. Add the respective field to the `InlineButton` https://github.com/tucnak/telebot/blob/59775c24a044783a06a5dd29ca48fa31a6818afd/markup.go#L267

2. Rename to `SwitchInlineQuery`